core/sql/parser/sqlparser.y

```diff
@@ -24909,7 +24946,6 @@ create_table_as_token: TOK_AS
 
 /* type pElemDDL */
 table_definition_body : table_element_list
-                      | like_definition
                       | external_table_definition
 
```

Comment:
This may be correct.  Not sure if the like options should be allowed for other create table productions.